settings/acsf/post-theme-deploy/clear-twig-cache.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Clears TWIG cache for a site after a theme deploy. Executes via drush alias.
```

Comment:
Add @todo Remove this script after https://www.drupal.org/node/2752961 is resolved and the corresponding Drupal core version is set as a minimum requirement for BLT.